src/TodoList.jsx

```diff
@@ -2,8 +2,8 @@ import { TodoListItem } from "./TodoListItem";
 
 const TodoList = ({ todoList }) => (
 	<ul className="list-wrapper">
-		{todoList.map((item) => (
-			<TodoListItem key={item.id} job={item} />
+		{todoList.map(({ id, ...item }) => (
```

Comment:
We should destructure everything rather than putting them in `...item` for better readability.

Reply:
done